package.json

```diff
@@ -6,30 +6,55 @@
         "prepublishOnly": "rm -rf lib; npm run transpile",
         "transpile": "mkdir -p lib; babel src --out-dir lib; node lib/js/generate.js",
         "test": "npm run transpile; nyc --reporter=text mocha --recursive --bail --require @babel/register/lib tests",
-        "postinstall": "npm run transpile"
+        "postinstall": "npm run transpile",
+        "lint": "eslint src tests && prettier --write src tests",
+        "lint:fix": "eslint --fix --cache src tests && prettier --write src tests",
+        "prettier": "prettier --check src tests",
+        "prepare": "husky install"
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/Exabyte-io/periodic-table.git"
+        "url": "https://github.com/Exabyte-io/periodic-table.js.git"
     },
     "main": "lib/js/index.js",
     "author": "Exabyte Inc.",
     "bugs": {
-        "url": "https://github.com/Exabyte-io/periodic-table/issues"
+        "url": "https://github.com/Exabyte-io/periodic-table.js/issues"
     },
     "license": "Apache-2.0",
-    "homepage": "https://github.com/Exabyte-io/periodic-table",
+    "homepage": "https://github.com/Exabyte-io/periodic-table.js",
     "dependencies": {
-        "@babel/cli": "7.0.0",
-        "@babel/core": "7.0.0",
-        "@babel/polyfill": "7.0.0",
-        "@babel/preset-env": "7.0.0",
-        "@babel/register": "7.0.0",
+        "@babel/cli": "^7.16.0",
+        "@babel/core": "^7.16.0",
```

Comment:
1. I suggest using an exact version here to avoid any potential problems - `7.16.0`
2. In the rest of our JS stack, the version of Babel is a bit older, we should keep this consistent across the codebase. Did we try and have the web-app working at 7.16? If not, maybe let's stick with the latest working version for now?

Reply:
Babel as a dependency is specified per package and is independent when invoking transpilation. We should have no issue specifying them separately, and it is one of the benefits of having separate packages. I agree we should be consistent, and we should be up-to-date if possible. 

We already updated the babel collection in MaDe.js to `7.16`. I am OK with exact version pinning but `^7.16.0` will only allow bugfix patches on `7.16`, which should remain API compatible.

I haven't vetted it 100% but there was no issue building/running the web app with `@babel/cli==7.16.0`, since it's already on `7.1.0`, I don't foresee many issues.

Reply:
OK, let's go with what we have then. We did have problems with using `^version` before, but it might be b/c we didn't have `package-lock.json`

Reply:
I meant just babel, but OK using the exact versions for the rest in https://github.com/Exabyte-io/periodic-table.js/pull/20/commits/91da66ca731e6f37ecf52639e9ac19b54dc1d95e

Reply:
I'm inclined to opt for pinned exact versions, and think we'll be OK with the versions specified so far.